x-pack/plugins/fleet/public/applications/fleet/sections/agents/agent_details_page/components/agent_details/agent_details_overview.tsx

```diff
@@ -57,6 +57,13 @@ export const AgentDetailsOverviewSection: React.FunctionComponent<{
               agent.policy_id || '-'
             ),
           },
+          {
+            title: i18n.translate('xpack.fleet.agentDetails.agentTypeLabel', {
+              defaultMessage: 'Agent type',
+            }),
+            // Title case. Leave first character unchanged. Lowercase the rest.
+            description: `${agent.type[0]}${agent.type.slice(1).toLowerCase()}`,
```

Comment:
@hbharding I wasn't sure how to present the agent type. I went with plain text in ~~Title~~ Sentence case. Let me know if it should be something different.